src/biguint.rs

```diff
@@ -434,6 +434,47 @@ impl One for BigUint {
 
 impl Unsigned for BigUint {}
 
+macro_rules! pow_impl {
+    ($unsigned:ty) => {
+         impl<'a> pow::Pow<$unsigned> for &'a BigUint {
+            type Output = BigUint;
+
+            #[inline]
+            fn pow(self, mut exp: $unsigned) -> Self::Output {
+                // square and multiply from num_traits::pow in the next branch
```

Comment:
Let's not immortalize the next branch in comments -- that stuff is obsolete.  If you're trying to give credit, then a note in your commit message is enough.

Reply:
:heavy_check_mark: